### DISCUSSION TITLE
Команда .npcbot order cast

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#290] [Q&A]
Всем привет!
Не уверен, что проблема в исходных файлах, поэтому пишу здесь. В общем у меня есть бот шаман ник Swart. Я отключил у него БЛ (Bloodlust - Жажда крови), т.к. это заклинание могут сдать на простом паке мобом. Использую команду .npcbot order cast swart bloodlust. Но сегодня после того как я отдаю команду в чат выходит сообщение "Swart doesn't have spell named 'bloodlust'!". Хотя ранее все работало.

**Comment by trickerer** [comment]
А ты не пробовал написать название спелла по-русски? С локализациями я эту команду не тестировал.

**Reply by xBarselonx** [reply]
Нет не пробовал. Сегодня попробую. Я так понимаю она должна выглядеть так - .npcbot order cast swart жажда_крови.

**Reply by xBarselonx** [reply]
Сегодня пробовал. Не работает.

**Reply by trickerer** [reply]
#294

**Reply by AlexPro54** [reply]
В прошлом году работало, а недавно, в какой-то момент времени перестали работать касты через .npcbot order cast.   Когда открываю через меню способности, то бот кастует, но это неудобно, хочется делать это через макрос. 
Пытаюсь в отладчике поймать строку кода где происходит пропуск команды, вроде это должно быть в HandleNpcBotOrderCastCommand.
Для примера ввел .npcbot order cast mage Снежная_буря
Парсер правильно определил номер класса 8 по классу и номер заклинания base_spell=42208. но вызова каста не происходит.
